src/content/upgrades/merge/eth-issuance/index.md

```diff
@@ -0,0 +1,100 @@
+---
+title: How The Merge impacts ether supply
+description: Breakdown on how The Merge will impact ether supply
+lang: en
+sidebar: true
+---
+
+# How The Merge impacts ether supply {#how-the-merge-impacts-ether-supply}
+
+<Card
+emoji=":chart_decreasing:">
+
+- Mining rewards ~13,000 ETH/day pre-merge
+- Staking rewards ~1,600 ETH/day pre-merge
+- **After The Merge, only the ~1,600 ETH per day will remain, dropping total new ETH issuance by ~90%**
+- <Emoji text=":fire:" size="1" /> The burn: At an average gas price of at least 16 gwei, at least 1,600 ETH is burned every day, which effectively brings net ETH inflation to zero or less post-merge.
+
+</Card>
+
+We can break the supply of ETH into two primary forces: issuance, and burn.
+
+The **issuance** of ETH is the process of creating ETH that did not previously exist on the balance sheet, while the **burning** of ETH refers to when existing ETH is permanently removed from circulation. The rate of each of these fluctuates based on a number of parameters, and the balance between them determines the resulting inflation/deflation rate of ether.
+
+The way ETH is issued will undergo some changes at time of The Merge. Currently, new ETH is issued from two main sources: the execution layer (i.e. Mainnet) and the consensus layer (i.e. Beacon Chain). After The Merge, issuance from the execution layer will go to zero. Let's break this down.
+
+## Pre-merge {#pre-merge}
+
+### Execution layer issuance {#el-issuance-pre-merge}
+
+Under proof-of-work, miners only interact with the execution layer, and are rewarded in the form of block rewards being the first to solve blocks. Since the [Constantinople upgrade](/history/#constantinople) in 2019 this reward has been 2 ETH per block. Miners are also rewarded for publishing [ommer](/glossary/#ommer) blocks, which are valid blocks that don't end up in the longest/canonical chain. These rewards max out at 1.75 ETH per ommer, and are _in addition to_ the reward issued from the canonical block. Mining is an economically intensive activity, which requires high levels of ETH issuance to sustain.
+
+### Consensus layer issuance {#cl-issuance-pre-merge}
+
+The [Beacon Chain](/history/#beacon-chain-genesis) has been live since 2020, which is secured by proof-of-stake validators, not miners. This chain was bootstrapped by Ethereum users depositing ETH one-way into a smart contract on Mainnet, which the Beacon Chain listens to, crediting the user with an equal amount on the new chain. Right now these validators are not processing transactions, and are essentially coming to consensus on the state of the validator pool itself.
```

Comment:
```suggestion
The [Beacon Chain](/history/#beacon-chain-genesis) went live in 2020. Instead of miners, it is secured by validators using proof-of-stake. This chain was bootstrapped by Ethereum users depositing ETH one-way into a smart contract on Mainnet, which the Beacon Chain listens to, crediting the user with an equal amount on the new chain. Until The Merge happens, the Beacon Chain's validators are not processing transactions and are essentially coming to consensus on the state of the validator pool itself.
```